deploy/helm/templates/servicemonitor.yaml

```
@@ -0,0 +1,23 @@
+{{- if eq .Values.serviceMonitor.enabled true }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "trivy-operator.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "trivy-operator.labels" . | nindent 4 }}
+spec:
+  namespaceSelector:
+    matchNames:
+    - {{ .Release.Namespace }}
```

```suggestion
  {{- if .Values.serviceMonitor.namespace }}
  namespaceSelector:
    matchNames:
    - {{ .Release.Namespace }}
  {{- end }}
```

We don't need to specify until the service is in another namespace.